lib/elixir/test/elixir/kernel/expansion_test.exs

```diff
@@ -595,6 +595,24 @@ defmodule Kernel.ExpansionTest do
         expand(quote(do: fn arg when make_ref() -> arg end))
       end
     end
+
+    test "in guards with bitstrings" do
+      message = ~r"cannot invoke remote function String.Chars.to_string/1 inside guards"
+
+      assert_raise CompileError, message, fn ->
```

Comment:
I did not include this before because I thought string interpolation in guards was going to be treated differently.

Reply:
Should we explicitly add a test for the cases that *work*? Which is literal and the results from expansion?

Reply:
The test suite has these ones (`bar/0` is defined at the top`defmacro bar, do: "bar"`):

https://github.com/elixir-lang/elixir/blob/7aaf02a28d3ace8bfc19c9960997efe82c108df4/lib/elixir/test/elixir/kernel/expansion_test.exs#L1993-L2007

I think that's enough, wdyt?

Reply:
Perfect, yes. Ship it!